# THE WILCOXON SIGNED-RANK TEST FOR LOCATION

The Wilcoxon signed-rank test is a non-parametric test used to compare the medians of two paired samples. It is a powerful and versatile test that can be used to detect even small differences between the medians of two samples.

#### 13.4.1
Sixteen laboratory animals were fed a special diet from birth through age 12 weeks. Their weight
gains (in grams) were as follows:

Can we conclude from these data that the diet results in a mean weight gain of less than 70 grams? Let
a $\alpha$ = .05, and find the p value.

In [1]:
library("MASS")
data <- c(63,68,79,65,64,63,65,64,76,74,66,66,67,73,69,76)

wilcox.test(data, mu = 70, alternative = "less", conf.int = TRUE, conf.level = 0.95)

Warning message in wilcox.test.default(data, mu = 70, alternative = "less", conf.int = TRUE, :
“cannot compute exact p-value with ties”
Warning message in wilcox.test.default(data, mu = 70, alternative = "less", conf.int = TRUE, :
“cannot compute exact confidence interval with ties”



	Wilcoxon signed rank test with continuity correction

data:  data
V = 48.5, p-value = 0.1622
alternative hypothesis: true location is less than 70
95 percent confidence interval:
     -Inf 70.99999
sample estimates:
(pseudo)median 
          68.5 


Based on the p-value of 0.1622, which is greater than the significance level of 0.05, we fail to reject the null hypothesis. This means that there is not enough evidence to conclude that the diet results in a mean weight gain of less than 70 grams based on the given data.

#### 13.4.2
Amateur and professional singers were the subjects of a study by Grape et al. (A-2). The researchers
investigated the possible beneficial effects of singing on well-being during a single singing lesson.
One of the variables of interest was the change in cortisol as a result of the signing lesson. Use the data
in the following table to determine if, in general, cortisol (nmol/L) increases after a singing lesson.
Let a $\alpha$ = .05. Find the p value.

In [2]:
data <- data.frame(
    before <- c(214,362,202,158,403,219,307,331),
    after <- c(232,276,224,412,562,203,340,313)
)

attach(data)
wilcox.test(before, after, paired = TRUE, alternative = "less", conf.int = TRUE, conf.level = 0.95)

Warning message in wilcox.test.default(before, after, paired = TRUE, alternative = "less", :
“cannot compute exact p-value with ties”
Warning message in wilcox.test.default(before, after, paired = TRUE, alternative = "less", :
“cannot compute exact confidence interval with ties”



	Wilcoxon signed rank test with continuity correction

data:  before and after
V = 9.5, p-value = 0.131
alternative hypothesis: true location shift is less than 0
95 percent confidence interval:
     -Inf 26.49996
sample estimates:
(pseudo)median 
     -24.46179 


Based on the p-value of 0.131, which is greater than the significance level of 0.05, we fail to reject the null hypothesis. This means that there is not enough evidence to conclude that cortisol generally increases after a singing lesson based on the given data.

#### 13.4.3
In a study by Zuckerman and Heneghan (A-3), hemodynamic stresses were measured on subjects
undergoing laparoscopic cholecystectomy. An outcome variable of interest was the ventricular end
diastolic volume (LVEDV) measured in milliliters. A portion of the data appear in the following table.
Baseline refers to a measurement taken 5 minutes after induction of anesthesia, and the term “5
minutes” refers to a measurement taken 5 minutes after baseline.
May we conclude, on the basis of these data, that among subjects undergoing laparoscopic
cholecystectomy, the average LVEDV levels change? Let a $\alpha$ = .01

In [3]:
data <- data.frame(
    before <- c(51.7,79.0,78.7,80.3,72.0,85.0,69.7,71.3,55.7,56.3),
    after <- c(49.3,72.0,87.3,88.3,103.3,94.0,94.7,46.3,71.7,72.3)
)

attach(data)
wilcox.test(before, after, paired = TRUE, alternative = "two.sided", conf.int = TRUE, conf.level = 0.99)

Warning message in wilcox.test.default(before, after, paired = TRUE, alternative = "two.sided", :
“cannot compute exact p-value with ties”
Warning message in wilcox.test.default(before, after, paired = TRUE, alternative = "two.sided", :
“cannot compute exact confidence interval with ties”



	Wilcoxon signed rank test with continuity correction

data:  before and after
V = 11.5, p-value = 0.1137
alternative hypothesis: true location shift is not equal to 0
99 percent confidence interval:
 -28.14995  15.99998
sample estimates:
(pseudo)median 
     -8.800001 


Based on the p-value of 0.1137, which is greater than the significance level of 0.01, we fail to reject the null hypothesis. This means that there is not enough evidence to conclude that there is a significant change in average LVEDV levels among subjects undergoing laparoscopic cholecystectomy based on the given data.